flink-end-to-end-tests/flink-sql-client-test/pom.xml

```diff
@@ -42,8 +42,55 @@ under the License.
 			<artifactId>scala-compiler</artifactId>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
```

Comment:
nit: Add a comment that a new jar has to be added here and in the bottom section.